winform/Forms/MainForm.Designer.cs

```diff
@@ -38,14 +38,17 @@ private void InitializeComponent()
             this.cUserItems = new System.Windows.Forms.CheckBox();
             this.tSettings = new System.Windows.Forms.TabControl();
             this.tabROMSettings = new System.Windows.Forms.TabPage();
-            this.cPatch = new System.Windows.Forms.CheckBox();
+            this.label9 = new System.Windows.Forms.Label();
```

Comment:
Would prefer that labels have proper names.

Reply:
i tried, but i'm not particularly motivated to give names to ui elements that aren't being altered programmatically.